autoreject/bayesopt.py

```diff
@@ -0,0 +1,60 @@
+"""Bayesian optimization according to Brochu, Cora, and de Freitas' tutorial
+http://haikufactory.com/files/bayopt.pdf
+"""
+
+# Authors: Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#          Mainak Jas <mainak.jas@telecom-paristech.fr>
```

Comment:
add alex passos to the authors as I started from his code:

http://atpassos.me/post/44900091837/bayesian-optimization